packages/core/admin/admin/src/components/LeftMenu/index.js

```diff
@@ -16,23 +16,14 @@ import {
   Button,
 } from '@strapi/parts';
 import ContentIcon from '@strapi/icons/ContentIcon';
-import { auth, usePersistentState } from '@strapi/helper-plugin';
+import { auth, usePersistentState, useAppInfos } from '@strapi/helper-plugin';
 import useConfigurations from '../../hooks/useConfigurations';
 
 const LeftMenu = ({ generalSectionLinks, pluginsSectionLinks }) => {
   const { menuLogo } = useConfigurations();
   const { push } = useHistory();
   const [condensed, setCondensed] = usePersistentState('navbar-condensed', false);
-
-  const userInfo = auth.getUserInfo();
-
-  let displayName;
-
-  if (userInfo && userInfo.firstname && userInfo.lastname) {
-    displayName = `${userInfo.firstname} ${userInfo.lastname}`;
-  } else {
-    displayName = get(userInfo, 'username', '');
-  }
+  const { userDisplayName } = useAppInfos();
```

Comment:
Should I do something else to see the user name?

![Screenshot 2021-09-03 at 13 27 28](https://user-images.githubusercontent.com/7756284/131998282-25bbb052-99d3-4cce-85b8-7f96b410065b.png)